### PR TITLE
10 project status updates

### LINKED ISSUE #10: Discussion: Project Status Codelist
In the current version of the project level schema, `phase` is an open codelist with the following codes:

* identification
* preparation
* construction
* completion

This differs from the project phases defined in [CoST guidance note 6](http://infrastructuretransparency.org/wp-content/uploads/2011/12/31_CoST_Guidance_Note_6_Proof.pdf) which uses 'implementation' rather than 'construction'.

The description of the implementation stage in the CoST guidance note covers the procurement and implementation of *planning* and *design* (as well as construction) contracts.

**Proposal:**

Align the codes and descriptions in the `phase` codelist with those in CoST guidance note 6:

Code | Description
------------ | -------------
identification | the decision to develop a project within the budget and programme of a project owner, which will be the public entity responsible for executing the budget
preparation | the feasibility study, environmental and social impact assessment, general scoping of the project, establishing the packaging and procurement strategy, preliminary statutory requirements on environmental and land impacts, and the resulting budget authorisation
implementation | the procurement and implementation of the planning, design and works according to the procurement strategy
completion | the handover of the assets and close-out activities with details of the final scope, cost and delivery time

**Discussion:**

* Can these definitions be clarified or improved? e.g. could the definition of 'implementation' include mention of supervision contracts
* How much of an issue is the difference in meaning between 'implementation' as a phase at the project level in the CoST IDS and 'implementation' as a stage of the contracting process in OCDS?

### PR DESCRIPTION
**Pull request by duncandewhurst**
Closes #10

### CODE REVIEW

File: schema/project-level/codelists/projectStatus.csv [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-cancelled,Cancelled,This project has been cancelled prior to close-out activities. Cancellation may have taken place at any point after identification.
+Code,Title,Description
+identification,Identification,Identification refers to the decision to develop a project within the budget and programme of a project owner (the public entity responsible for executing the budget).
+preparation,Preparation,"Preparation covers the feasibility study, environmental and social impact assessment, general scoping of the project, establishing the packaging and procurement strategy, preliminary statutory requirements on environmental and land impacts, and the resulting budget authorisation."
+implementation,implementation,"Implementation covers the procurement and implementation of the group of services, such as design and supervision, or works to be delivered under the project, including works or services performed by the procuring entity. This differs from the definition of implementation in OCDS which covers only the post-award phase of the contracting process."
```
Comment (jpmckinney) [inline schema/project-level/codelists/projectStatus.csv]: > Implementation covers the procurement and implementation of the group of works or services (such as design and supervision) to be delivered under the project, including works or services performed by the procuring entity.

I found the use of commas made this hard to parse.

> This differs from the definition of 'implementation' in OCDS, which covers the implementation but not the procurement.

Rather than introduce a new concept ("post-award phase"), I think it's clearer to refer to the concepts already covered in the first part of the definition.

File: schema/project-level/codelists/projectStatus.csv [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+completion,Completion,"Completion covers the handover of the assets and close-out activities with details of the final scope, cost, and delivery time."
+completed,Completed,"Close-out activities are complete, and this project is inactive."
```
[the remark this answers is not in the document]
Comment (duncandewhurst) [reply schema/project-level/codelists/projectStatus.csv]: Fixed in https://github.com/open-contracting/infrastructure/pull/99/commits/1378cf84d3910c8a01fb11e6e15d74c2517b282f

File: docs/reference/reference.md [hunk 3]
```diff
@@ -282,6 +282,8 @@ Selected codelists are displayed below.
 
 ```
 
+Projects with status 'completed' may be displayed as part of a list of archived projects.
```
[the remark this answers is not in the document]
Comment (duncandewhurst) [reply docs/reference/reference.md]: Fixed in https://github.com/open-contracting/infrastructure/pull/99/commits/1378cf84d3910c8a01fb11e6e15d74c2517b282f

File: schema/project-level/codelists/projectStatus.csv (hunk 1 above)
Comment (duncandewhurst) [reply schema/project-level/codelists/projectStatus.csv]: Fixed in https://github.com/open-contracting/infrastructure/pull/99/commits/1378cf84d3910c8a01fb11e6e15d74c2517b282f
